src/main/java/picard/fingerprint/ClusterCrosscheckMetrics.java

```diff
@@ -44,26 +44,52 @@
 import java.util.stream.Collectors;
 
 /**
- * Program to check that all (read-)groups within the set of input files appear to come from the same
- * individual. Can be used to cross-check libraries, samples, or files.
+ * <h4>Summary</h4>
+ * Clusters the results from a {@link CrosscheckFingerprints} run according to the LOD score. The resulting metric file
+ * can be used to assist diagnosing results from {@link CrosscheckFingerprints}. It clusters the connectivity graph between the
+ * different groups. Two groups are connected if they have a LOD score greater than the {@link #LOD_THRESHOLD}.
+ *
+ *
+ * <h4>Details</h4>
+ * The results of running {@link CrosscheckFingerprints} can be difficult to analyze, especially when many groups are
+ * related (meaning LOD greater than {@link #LOD_THRESHOLD}) in non-transitive manner (A is related to B, B is related to C,
+ * but A doesn't seem to be related to C.) {@link ClusterCrosscheckMetrics} clusters the metrics from {@link CrosscheckFingerprints}
+ * so that all the groups in a cluster are related to each other either directly, or indirectly (thus A, B and C would
+ * end up in one cluster.) Two samples can only be in two different clusters if all the samples from these two clusters
+ * do not get high LOD scores when compared to each other.
+ *
+ * <h4>Example</h4>
+ * <pre>
+ *     java -jar picard.jar ClusterCrosscheckMetrics \\ <hr />
```

Comment:
Same with the slashes ... and the hr tags also should go ... I guess they mean br rather than hr anyway.